src/main/java/jag/Command.java

```diff
@@ -0,0 +1,10 @@
+package jag;
```

Comment:
Would it be a better idea to organise all of the Command classes into a commands package instead?